packages/adapter-node/README.md

```diff
@@ -49,6 +49,10 @@ HOST=127.0.0.1 PORT=4000 node build
 
 You can specify different environment variables if necessary using the `env` option.
 
+## Deploying
+
+If you want to run the built app on another host then all is needed is the output directory and the `package.json` (for the `{"type": "module"}` setting).
```

Comment:
```suggestion
If you want to run the built app on another host, then all that is needed is the `build` directory and the `package.json` (for the `{"type": "module"}` setting).
```

We can mention `build` directly as that's fixed. I think we should move this under the middleware section too.

Reply:
thanks for the fixes. I didn't mention the `build` explicitly, because of the `out` setting, but happy to have the default here :)